projects/iot-cloud/stack.yml

```diff
@@ -6,19 +6,44 @@ services:
     volumes:
       - graphite:/opt/graphite/storage
       - graphite_conf:/opt/graphite/conf
+    environment:
+      VIRTUAL_HOST: 'graphite.beia.cloud,graphite.beia-telemetrie.ro,graphite.beia-consult.ro'
+      VIRTUAL_PORT: 80
     networks:
      - graphite
+     - proxy_net
 
   grafana:
     image: grafana/grafana
     environment:
-      GF_SECURITY_ADMIN_PASSWORD: '${GRAFANA_PASSWORD:-secret}'
+      GF_SECURITY_ADMIN_PASSWORD__FILE: '/run/secrets/grafana_password'
```

Comment:
Didn't tested but I guess there should be a single `_` (underscore) before `FILE`.

Reply:
It's not. https://github.com/grafana/grafana/blob/master/docs/sources/installation/docker.md#reading-secrets-from-files-support-for-docker-secrets